deploy/roles/postgresql-client/defaults/main.yml

```diff
@@ -6,8 +6,7 @@ postgres_install_dir: /usr/pgsql-9.5
 postgres_install_repository: True
 
 postgres_packages:
-  - postgresql95
-  - postgresql95-devel
+  - postgresql95-odbc
```

Comment:
@olivermel We should leave these packages here, and remove them from the other location.